controllers/drplacementcontrol_controller.go

```diff
@@ -744,6 +744,13 @@ func (r *DRPlacementControlReconciler) createDRPCInstance(
 		return nil, fmt.Errorf("failed to get DRPolicy %w", err)
 	}
 
+	if !drPolicy.ObjectMeta.DeletionTimestamp.IsZero() {
+		// If drpolicy is deleted then return
+		// error to fail drpc reconciliation
+		return nil, fmt.Errorf("drPolicy '%s' referred by the DRPC is deleted, DRPC reconciliation would fail",
```

Comment:
```suggestion
return nil, fmt.Errorf("drPolicy %q referred by the DRPC is deleted, DRPC reconciliation would fail",
			drpc.Spec.DRPolicyRef.Name)
```

use %q for %s to have a double quoted string.